src/main/java/bio/terra/common/logging/LoggingUtils.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.common.logging;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+public class LoggingUtils {
```

Comment:
can some of these classes be package private? It'd be helpful for services to only have access to the classes from this package that they need to do something with.

Reply:
Good idea – done where possible (I had to keep one public for mocking, but added @VisibleForTesting for clarity.